### PR TITLE
chore: Refactor FormField in Preview to use new styling utilities and tokens

### PR DESCRIPTION
**Pull request by mannycarrera4**
## Summary

Fixes: https://github.com/Workday/canvas-kit/issues/2398

## Release Category
Components

### Release Note
Update FormField in Preview to use our new styling utilities and tokens.

---

## Checklist

- [ ] MDX documentation adheres to Canvas Kit's [Documentation Guidelines](https://workday.github.io/canvas-kit/?path=/docs/guides-documentation-guidelines--page)
- [ ] Label `ready for review` has been added to PR

## For the Reviewer

- [ ] PR title is short and descriptive
- [ ] PR summary describes the change (Fixes/Resolves linked correctly)
- [ ] PR Release Notes describes additional information useful to call out in a release message or removed if not applicable
- [ ] Breaking Changes provides useful information to upgrade to this code or removed if not applicable

## Areas for Feedback? (optional)

- [ ] Code
- [ ] Documentation
- [ ] Testing
- [ ] Codemods

### CODE REVIEW

File: modules/preview-react/form-field/lib/FormFieldHint.tsx [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
 
 export const FormFieldHint = createSubcomponent('p')({
   displayName: 'FormField.Hint',
   modelHook: useFormFieldModel,
   elemPropsHook: useFormFieldHint,
 })<Omit<ExtractProps<typeof Subtext, never>, 'size'>>(
   ({children, ...elemProps}, Element, model) => {
-    const theme = useTheme();
-
+    console.log(formFieldHintStencil);
```
Comment (RayRedGoose) [inline modules/preview-react/form-field/lib/FormFieldHint.tsx:30]: Feels it should be removed :)

Comment (mannycarrera4) [reply modules/preview-react/form-field/lib/FormFieldHint.tsx:30]: who put that there?! señor pickle

Comment (RayRedGoose) [reply modules/preview-react/form-field/lib/FormFieldHint.tsx:30]: [!señor pickle Ogurchik](https://img.freepik.com/premium-vector/vintage-illustration-pickles-wear-hat-with-mustache-hold-stick_540637-597.jpg)

Comment (mannycarrera4) [inline modules/preview-react/form-field/lib/FormFieldHint.tsx:30]:
```suggestion
```

**Review by alanbsmith — APPROVED** [review]

**Pull request merged**